requirements.txt

```
@@ -0,0 +1,8 @@
+Django ==4.0.4
+django-cors-headers == 3.12.0
+mysqlclient == 2.1.0
+bcrypt == 3.2.2
+PyJWT == 2.4.0
+django-extensions == 3.1.5
```

- 필요한 패키지는 그때그때 해당 기능 PR에 추가해주세요.
- 띄어쓰기도 통일해주세요.
```suggestion
Django==4.0.4
django-cors-headers==3.12.0
mysqlclient==2.1.0
django-extensions==3.1.5
```